tslint.json

```diff
@@ -30,7 +30,7 @@
     ],
     "member-access": false,
     "member-ordering": [
-      true,
+      false,
```

Comment:
I actually do care about this ordering, so leave it as `true`, but let's also change the `order` property on line 35 to `"fields-first"` instead of what it is now.
So, in total, that would be:
```json
"member-ordering": [
    true,
    {
        "order": "fields-first"
    }
]
```